deploy/skaffold/Dockerfile

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 # This base image has to be updated manually after running `make build_deps`
```

Comment:
If you do want to rename build_deps &rarr; build-deps then
```suggestion
# This base image has to be updated manually after running `make build-deps`
```

Reply:
thanks